libs/net/dal/TNOContext.cs

```diff
@@ -85,11 +86,16 @@ public class TNOContext : DbContext
     #endregion
 
     #region ReportResults
-    public DbSet<CBRAReportTotalExcerpts> CBRAReportTotalExcerptsResults => Set<CBRAReportTotalExcerpts>();
-    public DbSet<CBRAReportStaffSummary> CBRAReportStaffSummaryResults => Set<CBRAReportStaffSummary>();
-    public DbSet<CBRAReportTotalsByProgram> CBRAReportTotalsByProgramResults => Set<CBRAReportTotalsByProgram>();
-    public DbSet<CBRAReportTotalsByBroadcaster> CBRAReportTotalsByBroadcasterResults => Set<CBRAReportTotalsByBroadcaster>();
-    public DbSet<CBRAReportTotalEntries> CBRAReportTotalEntriesResults => Set<CBRAReportTotalEntries>();
+    public IQueryable<CBRAReportStaffSummary> fn_cbra_report_staff_summary(DateOnly from, DateOnly to)
```

Comment:
We should use C# naming convention Pascal Case

Reply:
ok, I will update. thanks

Reply:
the method names have been updated. thanks